lib/common/run-app

```diff
@@ -36,9 +38,12 @@ else
 fi
 
 # Add packaged mono items to paths
-PATH="${MONO_PREFIX}/bin:${PATH}"
-LD_LIBRARY_PATH="${MONO_PREFIX}/lib:${LD_LIBRARY_PATH}"
+
 PKG_CONFIG_PATH="${MONO_PREFIX}/lib/pkgconfig:${PKG_CONFIG_PATH}"
+PATH="/opt/mono-sil/bin:${PATH}"
+LD_LIBRARY_PATH="/opt/mono-sil/lib:${LD_LIBRARY_PATH}"
+PKG_CONFIG_PATH="/opt/mono-sil/lib/pkgconfig:${PKG_CONFIG_PATH}"
+MONO_GAC_PREFIX="/opt/mono-sil:/usr"
```

Comment:
The above /opt/mono-sil paths would ignore the /usr/local path if that is the mono being used. It would be better to use the variable ${MONO_PREFIX} for PATH, LD_LIBRARY_PATH, PKG_CONFIG_PATH, and MONO_GAC_PREFIX.